scripts/travis_install

```diff
@@ -14,13 +14,13 @@ rm -rf $MINICONDA.sh
 
 python -V
 
-PINNED_PKGS=$(cat <<EOF
-binstar ==0.10.1
-EOF
-)
-echo -e "$PINNED_PKGS" > /home/travis/miniconda/conda-meta/pinned
+# PINNED_PKGS=$(cat <<EOF
+# binstar ==0.10.1
+# EOF
+# )
+# echo -e "$PINNED_PKGS" > /home/travis/miniconda/conda-meta/pinned
```

Comment:
Will this work uncommented, but with just deleting the binstar line? I think that would be preferable from a diff perspective.